### ISSUE TITLE
Feature Request: view current transfers

### ISSUE THREAD

**Issue by 1DontEx1st** [a1ex4/ownfoil#139]
Not important, but a Nice To Have.

**Comment by FoUStep** [comment]
+1, additionally see logged in/connected users.

**Comment by laleeroy** [comment]
> +1, additionally see logged in/connected users.

+1 to this, this would be great instead of just ip so we can monitor users that is  currently downloading

**Comment by a1ex4** [comment]
Cool idea !👍 Though it will be a bit of work to create a good UI around it.

**Comment by a1ex4** [comment]
Yes that's exactly what I'm planning to do. Each console can be uniquely identified so it will be automatically registered to your user in Ownfoil when logging in with Tinfoil. Then there would be a _Profile_ page where you could see the stats of your console, history of downloads, history of logins... The admin users could see the list of registered consoles in the user panel.

Definitely a cool feature that I have in mind since pretty much the start of the project, so I'll come around to it.
I just wish that Tinfoil would report more stats like Firmware version (to be able to filter on compatible game in the shop), storage...
